packages/sdk/src/sdk.ts

```diff
@@ -876,11 +875,45 @@ export class NxtpSdk {
   public async cancelExpired(cancelParams: CancelParams, chainId: number): Promise<providers.TransactionResponse> {
     const method = this.cancelExpired.name;
     const methodId = getRandomBytes32();
-    this.logger.info({ method, methodId, cancelParams, chainId }, "Method started");
+    this.logger.info({ method, methodId, chainId, cancelParams }, "Method started");
     const cancelRes = await this.transactionManager.cancel(chainId, cancelParams);
     if (cancelRes.isOk()) {
+      const response = cancelRes.value;
+      response.wait(1).then((cancelReceipt) => {
+        if (cancelReceipt.status === 0) {
+          this.logger.warn(
+            {
+              method,
+              methodId,
+              transactionId: cancelParams.txData.transactionId,
+              chainId,
+              transactionHash: cancelReceipt.transactionHash,
+            },
+            "Cancel transaction failed",
+          );
+          return;
+        }
+        this.logger.info(
+          {
+            method,
+            methodId,
+            chainId,
+            transactionId: cancelParams.txData.transactionId,
+            transactionHash: cancelReceipt.transactionHash,
+          },
+          "Mined cancel tx",
+        );
+        if (cancelParams.txData.sendingChainId === chainId) {
+          this.evts.SenderTransactionCancelled.post({
+            txData: cancelParams.txData,
+            relayerFee: cancelParams.relayerFee,
+            caller: cancelReceipt.from,
+            transactionHash: cancelReceipt.transactionHash,
+          });
+        }
```

Comment:
Receiver cancellations will be caught by the subgraph, but there is no good way (with current logic) to catch this on the subgraph on the receiver side